parsers/ua-parser-php-3/scripts/parse.php

```diff
@@ -53,7 +53,7 @@
             ],
             'platform' => [
                 'name'    => $r->os->family,
-                'version' => $r->ua->toVersion(),
+                'version' => null,
```

Comment:
Is version not available?

Reply:
That was the browser version. I have to check, if the platform version is available too,

Reply:
changed